Revit_Core_Adapter/Listener/Buttons/BHoMWiki.cs

```diff
@@ -36,7 +37,8 @@ public class BHoMWiki : IExternalCommand
 
         public Result Execute(ExternalCommandData commandData, ref string message, ElementSet elements)
         {
-            System.Diagnostics.Process.Start("https://github.com/BHoM/documentation/wiki");
+            var bHoMInformation = Query.Information();
+            System.Diagnostics.Process.Start($"{bHoMInformation.WikiLink}");
```

Comment:
`Process.Start` is quite common so I would shorten it :bowtie: 
```suggestion
            Process.Start($"{bHoMInformation.WikiLink}");
```